lib/features/upload/presentation/controller/upload_controller.dart

```diff
@@ -88,6 +88,7 @@ class UploadController extends BaseController {
                 (currentState) => currentState?.copyWith(uploadStatus: UploadFileStatus.uploadFailed));
             deleteFileUploaded(failure.uploadId);
             _showToastMessageWhenUploadAttachmentsFailure(failure);
+            consumeState(Stream.value((Left(failure))));
```

Comment:
how about to consume other `failure` state too?

Reply:
With upload only single errors from returned events are placed in ErrorAttachmentUploadState

Reply:
so put it out of if is also fair

Reply:
No, we have case Cancel upload